src/components/BankTransactions/BankTransactions.tsx

```diff
@@ -151,6 +151,15 @@ const BankTransactionsContent = ({
 
   useEffect(() => {
     if (JSON.stringify(inputFilters) !== JSON.stringify(filters)) {
+      /** @TODO - Do we want to default filters to fetch categorized and uncategorized transactions when categorization is disabled? */
+      /* if (!categorizationEnabled) {
+        setFilters({
+          ...filters,
+          ...inputFilters,
+          categorizationStatus: DisplayState.all,
+        })
+      }
+      else if (... */
```

Comment:
Do we want to fetch "categorized" and "uncategorized" transactions when bookkeeping is active or categorization is disabled?